http-gateways/PATH_GATEWAY.md

```diff
@@ -238,7 +239,7 @@ Optional, `format=<format>` can be used to request specific response format.
 
 This is a URL-friendly alternative to sending
 `Accept: application/vnd.ipld.<format>` header, see [`Accept`](#accept-request-header)
-for more details.
+for more details. In case of `application/x-tar`, `<format>` is `tar`.
```

Comment:
Let's separate this and clarify, otherwise people will be coming up with `application/vnd.ipld.<format>` → `application/vnd.ipld.tar` ;) 

```suggestion
for more details. 

In case of `Accept: application/x-tar`, the `?format=` equivalent is `tar`.
```